src/Create/Create.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react";
+
+type CreateProps = {
+  //
+};
+
+class Create extends Component<CreateProps, any> {
+  public render() {
+    return <div>Create</div>;
+  }
+}
```

Comment:
React supports backwards compatibility but we should be should using the newest suggested style that they have for creating components. you can refer to their docs on what they suggest to write.

this is what it would look like in the newer style.

```tsx

const Create = (props: CreateProps) => {
    return <div>Create</div>;
}

// or

function Create(props: CreateProps) {
    return <div>Create</div>;
}
```